behave/requirements.txt

```diff
@@ -8,4 +8,7 @@ requests==2.26.0
 pre-commit==2.17.0
 axe-selenium-python==2.1.6
 #need to use cla_common to check the availability of slots for the creation of callbacks
-git+https://github.com/ministryofjustice/cla_common.git@0.3.18#egg=cla_common
\ No newline at end of file
+git+https://github.com/ministryofjustice/cla_common.git@0.3.18#egg=cla_common
```

Comment:
might make sense to update this to the tag I created for the disregards

Reply:
https://github.com/ministryofjustice/cla_common/releases/tag/0.3.19

Reply:
I have tested it on my local using the 0.3.19 tag but don't want to commit it as you have already committed to it master